lib/common/util.js

```diff
@@ -360,6 +362,14 @@ function makeAuthenticatedRequestFactory(config) {
     if (stream) {
       return stream;
     }
+
+    return {
+      abort: function() {
+        if (activeRequest_) {
+          activeRequest_.abort();
```

Comment:
I would like to, but `activeRequest_` comes to exist asynchronously via `onAuthenticated`